core/test/user-flow-test.js

```diff
@@ -331,38 +326,38 @@ describe('UserFlow', () => {
         },
       };
 
-      /** @type {LH.UserFlow.GatherStep[]} */
```

Comment:
why not
```
// @ts-expect-error - incomplete mock.
/** @type {LH.UserFlow.GatherStep[]} */
```

or whatever? This file can also just be removed from the tsconfig if mocking continues to make it too annoying to maintain the types.